docs/guides/getting-started/index.html

```diff
@@ -0,0 +1,81 @@
+---
+title: Getting Started
+---
+{% extends 'guide.njk' %}
+{% block content %}
+  <section>
+    <p>
+      This guide is intented to help developers initialize a new application.
+    </p>
+  </section>
+
+  <section><!-- Install from NPM -->
+    <h2 class="hxSectionTitle" id="install">Step 1: Install via npm or Yarn</h2>
+    <p>
+      HelixUI is registered as a package on <a href="https://www.npmjs.com/">npm</a>.
+      You can install the latest version with the npm CLI command:
+    </p>
+    {% code "bash" %}
+      npm install --save-dev helix-ui
+    {% endcode %}
+
+    <p>
+      Alternatively, you can use the <a href="https://yarnpkg.com">Yarn</a>
+      CLI command:
+    </p>
+    {% code "bash" %}
+      yarn add helix-ui
+    {% endcode %}
+
+    <p>
+      Either strategy installs HelixUI in the <code>node_modules/</code> directory.
+    </p>
+  </section>
+
+  <section><!-- Choose a Layout -->
+    <h2 class="hxSectionTitle" id="choose-layout">Step 2: Choose a Layout</h2>
+    <p>
+      Building a Helix application begins with the layout.
+      HelixUI provides two supported layouts that you can choose from:
+    </p>
+    <ol>
+      <li>
+        <a href="components/layouts/#vertical-layout">Vertical Layout</a>
+        <ul>
+          <li>Content grows downward, along y-axis</li>
+          <li>Footer is pushed down as content grows</li>
```

Comment:
Will have a better sense of how to word this based on the comment above. Could be "As content populates a page, the placement of the footer changes based on the amount of content."?

Reply:
This seems wordy and vague with regards to the position of the footer.